tests/integration/models/reddit/test_rules.py

```diff
@@ -0,0 +1,194 @@
+from unittest import mock
+import pytest
+
+from praw.exceptions import RedditAPIException, ClientException
+from praw.models import Rule
+
+from ... import IntegrationTest
+
+
+class TestRule(IntegrationTest):
+    @property
+    def subreddit(self):
+        return self.reddit.subreddit(pytest.placeholders.test_subreddit)
+
+    def test_add_rule(self):
+        self.reddit.read_only = False
+        with self.recorder.use_cassette("TestRule.test_add_rule"):
+            rule = self.subreddit.rules.mod.add(
+                "PRAW Test",
+                "all",
+                description="Test by PRAW",
+                violation_reason="PTest",
+            )
+            assert rule.short_name == "PRAW Test"
```

Comment:
While it's not something that needs to be changed, I typically prefer the `assert` statements to be outside of the betamax context manager so that we can be sure they don't issue a network request (unless we explicitly expect them to).